### PR TITLE
[TS_SELENIUM] Create the "NotificationCenter" pageobject

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
Signed-off-by: Ihor Okhrimenko <iokhrime@redhat.com>

### What does this PR do?
Create the "NotificationCenter" pageobject

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/15903

### CODE REVIEW

**Review by ScrewTSW — APPROVED** [review]
LGTM - nice work with extracting the notification center methods

File: tests/e2e/pageobjects/ide/NotificationCenter.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    async clickCloseAllNotificationsButton(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        Logger.debug(`NotificationCenter.clickCloseAllNotificationsButton`);
+
+        const closeAllButtonLocator: By = By.css('.theia-notification-center-header-actions .clear-all');
+
+        await this.driverHelper.waitAndClick(closeAllButtonLocator, timeout);
+    }
+
+    async waitClosing(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        Logger.debug(`NotificationCenter.close`);
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/ide/NotificationCenter.ts]: The method name is `waitClosing`, so "Logger.debug(`NotificationCenter.waitClosing`);" looks more consistent, IMHO

File: tests/e2e/pageobjects/ide/NotificationCenter.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    async waitClearNotificationsList(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        Logger.debug(`NotificationCenter.waitClearNotificationsList`);
+
+        const notificationLocator: By = By.css('.theia-notification-center .theia-notification-list > *');
+
+        await this.driverHelper.waitDisappearance(notificationLocator, timeout);
+    }
+
+    async clearAll(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/ide/NotificationCenter.ts]: The method is not just clear all, but **close** all the notifications. IMHO, the name `closeAll()` could be more relevant here.

**Review by dmytro-ndp — APPROVED** [review]
Looks good!

**Pull request merged**